src/style.css

```diff
@@ -0,0 +1,135 @@
+.page {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  --blue-color: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  padding: 14px;
+}
+
+.card__img {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 15px 5px;
+}
+
+.img {
+  object-fit: cover;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  padding-top: 30px;
+  padding-bottom: 6px;
+  color: #000;
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: 500;
+}
+
+.card__id {
+  width: 100%;
+  padding-bottom: 15px;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__rating {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 20px;
+}
+
+.rating {
+  display: flex;
+}
+
+.rating__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.rating__star--bad {
+  background-image: url("./images/star.svg");
+}
+
+.rating__star--good {
+  background-image: url("./images/star-active.svg");
+}
+
+.reviews {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__price {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+  padding-bottom: 20px;
+}
+
+.price-title {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.price-value {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.bottom-buy {
+  display: flex;
+  width: 100%;
+  height: 40px;
+  border-radius: 5px;
+}
+
+.card__bottom-buy {
+  background-color: var(--blue-color);
+  color: #fff;
+}
+
+.card__bottom-buy:hover {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid var(--blue-color);
+  color: var(--blue-color);
+}
+
+.bottom-buy__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: inherit;
+  height: inherit;
+  border-radius: inherit;
```

Comment:
Let's not inherit everything but just get rid of that 'div' wrapper. You may style your `.bottom-buy__link` to make it look like a proper button. Let's try